fhservices.cfg.j2

```diff
@@ -141,3 +146,12 @@ define service {
        notes The mongodb replica set may not be functioning properly.  There should be one and only one primary member and zero or more secondary members
        contact_groups rhmapadmins
 }
+
+define service {
+       service_description Pod::Memory Usage
```

Comment:
Probably, I was basing it on what the disk check called it, but its pod containers really.